app/controllers/evidence_controller.rb

```diff
@@ -122,11 +122,14 @@ def destroy
           # 2. from the node evidence tab
           # 3. from the evidence show page itself (under node)
           # When using redirect_back in case 3, we find an evidence not found error,
-          # since the evidence does not exist anymore. That's why we check the 'Referer' here:
-          if request.headers['Referer'] == project_node_evidence_url(current_project, @node, @evidence)
+          # since the evidence does not exist anymore. That's why we check the referrer here.
```

Comment:
Aaron, I understand this code is to keep showing the evidence tab when destroying evidence.

When destroying from nodes evidence tab, seems like we were not doing it, even Iif we redirected when creating/updating.

Maybe that decision was taken to avoid complicating this too much?
If we cannot find a simper solution, I would just leave this as it was and avoid returning to evidence tab on destroy

I think this is a case of https://stackoverflow.com/questions/51054260/rails-5-redirect-back-with-url-query-parameter-option

Reply:
> When destroying from nodes evidence tab, seems like we were not doing it, even Iif we redirected when creating/updating.

Fixed this. I simply removed an `elsif` condition since it pretty much does the same as the `else` case.

> I think this is a case of https://stackoverflow.com/questions/51054260/rails-5-redirect-back-with-url-query-parameter-option

This is exactly it. `redirect_back` would have been a good choice but we can't use it here as we need to attach the tab param.